src/DynamixelMotor.cpp

```diff
@@ -121,3 +127,39 @@ uint16_t DynamixelMotor::currentPosition()
 	return currentPosition;
 }
 
+DynamixelStatus DynamixelMotor::getCurrentPosition(uint16_t &pos)
+{
+	return read(DYN_ADDRESS_CURRENT_POSITION, pos);
+}
+
+DynamixelStatus DynamixelMotor::setComplianceMargins(byte cw_margin, byte ccw_margin, byte cw_slope, byte ccw_slope)
+{
+	DynamixelStatus status;
```

Comment:
mStatus is private to DynamixelDevice, so it looks like I need to call status() to get it (suggest changing that to getStatus() ). This extra function call seems less efficient than just passing through the return values from read() and write(), if we're nitpicking :)